cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -111,7 +110,7 @@ function error_exit
   # TODO: add possibility to disable this behavior
   # wait logs flush before signaling the failure
   sleep 10
-  aws --region ${!region} ec2 terminate-instances --instance-ids ${!instance_id}
+  shutdown
```

Comment:
shall we execute `shutdown -h now` otherwise in case of logged in users it will wait for a default timeout?

Reply:
indeed this was necessary